openreview/conference/matching.py

```diff
@@ -242,7 +252,20 @@ def setup(self, affinity_score_file = None, tpms_score_file = None):
                 'readers': {'values': [CONFERENCE_ID, PROGRAM_CHAIRS_ID]},
                 'writers': {'values': [CONFERENCE_ID]},
                 'signatures': {'values': [CONFERENCE_ID]},
-                'content': {}
+                'content': {
+                    'title': {
+                        'value-regex': '.{1,250}',
+                        'required': True,
+                        'description': 'Title of the configuration.',
+                        'order': 1
+                    },
+                    'assigned_groups': {
```

Comment:
I'm sorry, the names of the fields are 'assignedGroups' and 'alternateGroups', there is an example here:

https://openreview.net/notes?invitation=thecvf.com/ICCV/2019/Workshop/Pre-Reg/-/Paper_Assignment